### PR TITLE
More zip codes

### PR DESCRIPTION
**Pull request by aarongray**
So it turns out that TZip was missing about 5000 zip codes. This should bring TZip up to speed with the current set of zip codes in the US.

It also codifies the ruby version since it was absent previously and that meant trouble for new people pulling down this codebase.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,9 +1,10 @@
 source "http://rubygems.org"
+ruby "1.9.3"
```
Comment (ryoung) [inline Gemfile:2]: ruby 1.9?

Comment (aarongray) [reply Gemfile:2]: Yeah this appears to be the version of ruby that this library was built with. Modern versions of ruby got me lost in dependency hell, and older versions of ruby segfaulted when installing the gems. I had to experiment and try quite a few different versions of ruby in order to find the one they apparently built this gem with. :laughing:

Comment (ryoung) [reply Gemfile:2]: wow, shows how old this gem is

**Review by ryoung — APPROVED** [review]

File: Gemfile (hunk 1 above)
Comment (aarongray) [reply Gemfile:2]: for sure, also helps explain how so many zip codes were missing from their database

**Pull request merged**